src/proof/freeze.rs

```diff
@@ -442,7 +445,7 @@ mod test {
         let fee = Amount::from(5u64);
         let fee_keypair = UserKeyPair::generate(rng);
         let freeze_keypair = FreezerKeyPair::generate(rng);
-        let recv_memos_ver_key = KeyPair::generate(rng).ver_key();
+        let recv_memos_ver_key = schnorr::KeyPair::generate(rng).ver_key();
```

Comment:
Why was it directly `KeyPair` before?

Reply:
because we have the `pub type KeyPair = schnorr::KeyPair<CurveParam>;` in `types.rs`